nerium/contrib/resultset/takei.py

```diff
@@ -0,0 +1,54 @@
+import os
+import re
+
+import records
+from nerium.contrib.resultset.sql import SQLResultSet
+
+
+class TakeiResultSet(SQLResultSet):
+    """ Handle table name substitution in takei queries
+    """
+    # TODO: Remove this module as soon as we can
+    # replace legacy 'takei-web-app' MySQL instance
+
+    def table_substitution(self, db, client_id):
+        table_list = db.get_table_names()
+        with open(self.get_query_path(), 'r') as _query_file:
+            sql_query = _query_file.read()
+        if '_TABLE' not in sql_query:
+            return sql_query
+        table_tokens = re.findall(r'\w+_TABLE\b', sql_query)
+        table_norm = [i.lower().replace('_table', '') for i in table_tokens]
+        table_sub = ['`{}_{}`'.format(client_id, i) for i in table_norm]
+        for table in table_sub:
+            if table not in table_list:
+                return ('missing', table)
+        table_map = list(zip(table_tokens, table_sub))
+        for map in table_map:
+            sql_query = sql_query.replace(map[0], map[1])
+        return sql_query
+
+    def result(self):
+        backend_path = self.get_query_path().parent
+        backend_template = self.backend_lookup(backend_path)
+        takei_pwd = os.getenv('MYSQL_PWD', None)
+        backend = backend_template.format(password=takei_pwd)
+        db = records.Database(backend)
+
+        client_id = self.kwargs.get('client_id', 0)
```

Comment:
Does this seem like a reasonable thing to do when `client_id` is not supplied as a kwarg? I could change this to some other value, but it at least allows table_substitution to run...